test/test/test.js

```diff
@@ -18,14 +18,19 @@ describe('Bridge', function () {
   const polkadotRecipient = "0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d";
   const polkadotRecipientSS58 = "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY";
 
-  const ETH_ASSET_ID = "0x00"
+	const subNullRecipient = "0x0000000000000000000000000000000000000000";
```

Comment:
can you please share a script to run these tests?

Reply:
Please see /test/setup/Run.